client/elife-theme/src/index.js

```diff
@@ -5,49 +5,49 @@ export default {
   /* Colors */
   colorBackground: '#fff',
```

Comment:
Why is this `colorBackground` here? I haven't seen it in the styleguide nor the VARIABLES.md.